content/en/blog/2025/2025-08-30-pachli-2-16-0.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Pachli 2.16.0 released"
+linkTitle: "Pachli 2.16.0 released"
+date: 2025-08-30
+author: Nik Clayton ([@nikclayton](https://mastodon.social/@nikclayton))
```

Comment:
This could also reference the Pachli instead. I don't have a strong preference either way.